app/main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import NotWearingMaskError
+from app.errors import VaccineError
+
+
+def go_to_cafe(friends: list[dict], cafe: Cafe) -> str:
+
+    friends_has_access = True
+    masks_to_buy = 0
+
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except NotWearingMaskError:
+            friends_has_access = False
```

Comment:
This is redundant

Reply:
You can just check that if masks_to_buy == 0

Reply:
done